resources/rpm/spec.erb

```diff
@@ -9,7 +9,7 @@
 %define __spec_clean_pre true
 
 # Use gzip payload compression
```

Comment:
I think we should update this comment to something more generic

Reply:
fixed